bloom/generators/rpm/templates/catkin/template.spec.em

```diff
@@ -41,7 +41,6 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
     -USYSCONF_INSTALL_DIR \
     -USHARE_INSTALL_PREFIX \
     -ULIB_SUFFIX \
-    -DCMAKE_INSTALL_LIBDIR="lib" \
```

Comment:
If this change is also applied to catkin packages, won't it cause a regression for libccd (assuming it is still being vendored in ROS)?

Are there other ROS one packages that follow a third party vendor pattern even if they don't use the ROS 2 vendor conventions?

Reply:
My guess is that a similar approach to the one we're taking in ROS 2 was taken in ROS 1 to mitigate the issue that originally motivated that change, and I just wasn't aware of it because it was debian-specific.

I'd be more concerned if we were actually building ROS 1 packages from these spec files, but as of today, we're not.

tl;dr - if it's still a problem, we'll need to find a better solution. This just wasn't the right thing to do.

Reply:
Thanks for adding more context. I wanted to check my assumption that there is no other change which fixes the issue #372 and that has borne out.